README.md

```diff
@@ -1,3 +1,64 @@
+# Pomodoro Project
+
+<img src="https://user-images.githubusercontent.com/69774407/119630519-b11e2480-be17-11eb-8f89-f1179db1948e.png" width="100" height="100">
+
+## The Pomodoro Technique is a time management method.
+#### The technique uses a timer to break down work into intervals, traditionally 25 minutes in length, separated by short breaks. Each interval is known as a pomodoro, from the Italian word for 'tomato', after the tomato-shaped kitchen timer that Cirillo used as a university student.
```

Comment:
These should be regular paragraph text and not header text.

If you copied and pasted this code from another website, this counts as plagiarism. In order to not plagiarize, you must mark this text as a quote and reference where you got it from.

Alternatively, you can paraphrase using your own words.